src/lib/ldap/conf.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+/**
+ * $Id$
+ * @file lib/ldap/conf.h
+ * @brief Conf parser sections common to rlm_ldap and ldap sync
+ *
+ * @copyright 2022 The FreeRADIUS Server Project.
+ */
+
+#include <freeradius-devel/ldap/base.h>
+
+static CONF_PARSER fr_ldap_sasl_mech_static[] = {
```

Comment:
Can you still this all in conf.c and then add `extern` declarations in conf.h, that way we don't have the bytecode being duplicated for every use.